docs/conf.py

```diff
@@ -112,64 +113,51 @@
 
 
 # -- Options for LaTeX output ---------------------------------------------
-
-latex_elements = {
-    # The paper size ('letterpaper' or 'a4paper').
-    #
-    # 'papersize': 'letterpaper',
-
-    # The font size ('10pt', '11pt' or '12pt').
-    #
-    # 'pointsize': '10pt',
-
-    # Additional stuff for the LaTeX preamble.
-    #
-    # 'preamble': '',
-
-    # Latex figure (float) alignment
-    #
-    # 'figure_align': 'htbp',
-}
+#
+# These options may be included in the dictionary below:
+#   The paper size ('letterpaper' or 'a4paper').
+#   'papersize': 'letterpaper',
+#
+#   The font size ('10pt', '11pt' or '12pt').
+#   'pointsize': '10pt',
+#
+#   Additional stuff for the LaTeX preamble.
+#   'preamble': '',
+#
+#   Latex figure (float) alignment
+#   'figure_align': 'htbp',
+latex_elements = {}
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
-latex_documents = [
-    (master_doc, 'pysatModels.tex', u'pysatModels Documentation',
-     u'Angeline G Burrell, Russel Stoneback, and Jeffery Klenzing', 'manual'),
-]
+latex_documents = [(master_doc, 'pysatModels.tex', u'pysatModels Documentation',
+                    author, 'manual')]
 
 
 # -- Options for manual page output ---------------------------------------
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [
-    (master_doc, 'pysatModels', u'pysatModels Documentation',
-     [author], 1)
-]
-
+man_pages = [(master_doc, project, u'pysatModels Documentation',
+              [author], 1)]
 
 # -- Options for Texinfo output -------------------------------------------
 
 # Grouping the document tree into Texinfo files. List of tuples
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
-texinfo_documents = [
-    (master_doc, 'pysatModels', u'pysatModels Documentation',
-     author, 'pysatModels', 'Tools for Model-Data analysis.',
-     'Space Physics'),
-]
-
-
+texinfo_documents = [(master_doc, project, u'pysatModels Documentation',
+                      author, project, 'Tools for Model-Data analysis.',
```

Comment:
`description` is only used once that I found, but could be as a variable up top along with title for maintenance